hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -176,8 +176,8 @@ private synchronized void connectToDatanode(DatanodeDetails dn)
 
     // Add credential context to the client call
     if (LOG.isDebugEnabled()) {
-      LOG.debug("Nodes in pipeline : {}", pipeline.getNodes());
-      LOG.debug("Connecting to server : {}", dn.getIpAddress());
+      LOG.debug("Nodes in pipeline : {}, connecting to server : {}",
+          pipeline.getNodes(), dn.getIpAddress());
```

Comment:
Let's swap the parts of the message to make it easier to see the selected node.  List of nodes in pipeline can be longer.  Also, rely on `DatanodeDetails.toString()` to get more information (hostname and UUID in addition to IP address).

```suggestion
      LOG.debug("Connecting to server: {}; nodes in pipeline: {}",
          dn, pipeline.getNodes());
```

Reply:
Thanks @adoroszlai  for the comment and review.
I have updated it.